bonus.md

```diff
@@ -3,11 +3,16 @@
 Edit this file to answer the following questions. Give examples. You may need to do a bit of research. :)
 
 1. Why is Hash.fetch useful?
+  It is useful because you can search through the Hash in any order.  If the key isn't found, an error will be returned which can be useful for you to know where the program is failing.
```

Comment:
Using bracket notation also doesn't require you to access in any particular order -- using a hash in general makes it so that you don't need to pass the args in order.